apps/pie-docs/src/assets/styles/components/_homePage.scss

```diff
@@ -0,0 +1,110 @@
+@use '../fozzie-config' as f;
+
+.c-hero {
+    min-height: 100%;
+    display: grid;
+
+    // helps scale the razor with the page
+    grid-template-rows: minmax(0, 300px) minmax(0, 100px) auto;
+    background-color: f.$color-background-subtle;
+
+    @include f.media('>=mid') {
+        grid-template-rows: minmax(0, 200px) minmax(0, 200px) auto;
+    }
+
+    @include f.media('>=wide') {
+        grid-template-rows: minmax(0, 220px) minmax(0, 240px) auto;
+    }
+}
+
+.c-heroBanner,
+.c-heroContent {
+    padding-inline: f.$spacing-d;
+
+    @include f.media('>=narrowMid') {
+        padding-inline: f.spacing(e);
+    }
+
+    @include f.media('>=mid') {
+        padding-inline: f.spacing(i);
+    }
+
+    .c-content .c-content-container {
+        @include f.media('<mid') {
+            padding-top: 0 !important;
+        }
+    }
+}
+
+.c-heroBanner {
+    background-color: f.$color-background-subtle;
+}
+
+.c-heroContent {
+    background-color: f.$color-background-default;
+}
+
+.c-heroBanner-heading {
+    grid-row: 1 / span 2;
+    display: flex;
+    flex-direction: row;
+    min-height: 400px;
+    gap: f.$spacing-e;
+    justify-content: space-between;
+
+    .c-heroBanner-heading-text {
+        width: 100%;
+        padding-top: f.spacing(g);
+
+        @include f.media('>=mid') {
+            width: 50%;
+            max-width: 500px;
+            padding-top: 0;
+            align-self: center;
+        }
+
+        @include f.media('>=wide') {
+            align-self: flex-start;
+            padding-top: calc(f.spacing(e) + f.spacing(i)); // 88px
+        }
+    }
+
+    .o-btn {
+        margin-top: f.spacing(f);
+    }
+
+    .c-heroBanner-image {
+        align-self: center;
+        z-index: 1;
+        width: 50%;
+
+        img {
+            object-fit: contain;
+            width: 100%;
+            max-height: 100%;
+            min-width: 350px;
+            position: relative;
+        }
+    }
+}
+
+.c-razor {
```

Comment:
is this going to be reused anywhere? if so, probably best in a separate dedicated file. If not, perhaps `c-homeHero-razor`

Reply:
I'm not aware of it being reused anywhere - will change to `c-homeHero-razor` for now 👍